src/abstractdataframe/show.jl

```diff
@@ -57,6 +57,7 @@ end
 ourshowcompact(io::IO, x::Any) = show(IOContext(io, :compact=>true), x) # -> Void
 ourshowcompact(io::IO, x::AbstractString) = escape_string(io, x, "") # -> Void
 ourshowcompact(io::IO, x::Symbol) = ourshowcompact(io, string(x)) # -> Void
+ourshowcompact(io::IO, x::Nothing) = ourshowcompact(io, "") # -> Void
```

Comment:
Couldn't we just do nothing and return `nothing`? Calling `ourshowcompact` on an empty strings sounds overkill for a no-op.

Reply:
That's a good idea. We don't need to print anything, just define `strlen` to be zero.